prac_05/hex_colours.py

```diff
@@ -0,0 +1,11 @@
+COLOUR_AND_HEX = {"black": "#000000", "ghostwhite": "#f8f8ff", "gray": "#bebebe", "lawngreen": "#7cfc00",
+                  "light": "	#eedd82", "linen": "#faf0e6", "medium": "#66cdaa", "mintcream": "#f5fffa",
+                  "navyBlue": "#000080", "olivedrab": "#6b8e23"}
+print(COLOUR_AND_HEX.keys())
+colour = input("Hex for colour>>>")
```

Comment:
This is backwards - the user should enter the colour name, not the hex.
"AND" is not as good as "TO" in directory naming.